topics/assembly/tutorials/vgp_workflow_training/tutorial.md

```diff
@@ -58,102 +58,179 @@ GWS facilitates analysis repeatability, while minimizing the number of manual st
->
->    - From **Rules** menu select `Add / Modify Column Definitions`
->       - Click `Add Definition` button and select `List Identifier(s)`: column `A`
->       - Click `Add Definition` button and select `URL`: column `B`
->       - Click `Add Definition` button and select `Type`: column `C`
->       - Click `Add Definition` button and select `Group Tag`: column `D`
->       - Click `Add Definition` button and select `Collection Name`: column `E`
->    - Click `Apply` and press <kbd>Upload</kbd>
->
-{: .hands_on}
+The following steps use PacBio {HiFi} and Illumina {Hi-C} data from baker's yeast ([*Saccharomyces cerevisiae*](https://en.wikipedia.org/wiki/Saccharomyces_cerevisiae)). The tutorial represents trajectory **B** from Fig. 1 above. For this tutorial, the first step is to get the datasets from Zenodo. Specifically, we will be uploading two datasets:
 
+1. A set of PacBio {HiFi} reads in `fasta` format
+2. A set of Illumina {Hi-C} reads in `fastqsanger.gz` format
 
-> <details-title>Working with your own data</details-title>
->
-> If working on a genome other than the example yeast genome, you can upload the VGP data from the [VGP/Genome Ark AWS S3 bucket](https://genomeark.s3.amazonaws.com/index.html) as follows:
->
-> > <hands-on-title>Import data from GenomeArk</hands-on-title>
-> >
-> > 1. Open the file {% icon galaxy-upload %} __upload__ menu
-> > 2. Click on **Choose remote files** tab
-> > 3. Click on the **Genome Ark** button and then click on **species**
-> {: .hands_on}
->
-> You can find the VGP data following this path: `/species/${Genus}_${species}/${specimen_code}/genomic_data`. Inside a given datatype directory (*e.g.* `pacbio`), select all the relevant files individually until all the desired files are highlighted and click the <kbd>Ok</kbd> button. Note that there may be multiple pages of files listed. Also note that you may not want every file listed.
->
-> {% snippet faqs/galaxy/collections_build_list.md %}
->
-{: .details}
+## Uploading `fasta` datasets from Zenodo
+
+The following two steps demonstrate how to upload three PacBio {HiFi} datasets into you Galaxy history.
+
+### **Step 1**: Copy the following URLs into clipboard
+(you can do this by clicking on {% icon copy %} button in the right upper corner of the box below. It will appear if you mouse over the box.)
+
+```
+https://zenodo.org/record/6098306/files/HiFi_synthetic_50x_01.fasta
+https://zenodo.org/record/6098306/files/HiFi_synthetic_50x_02.fasta
+https://zenodo.org/record/6098306/files/HiFi_synthetic_50x_03.fasta
+```
+
+### **Step 2**: Upload datasets into Galaxy
+
+These datasets are in `fasta` format. Upload by following the steps shown in the figure below.
+
+------
+
+![Figure 2: Uploading fasta files in Galaxy]({{site.baseurl}}/faqs/galaxy/images/upload_fasta_via_url.png "Here we upload three fasta files. Compressed (.gz or .bz2) datasets are uploaded in exactly the same fashion by selecting an appropriate datatype (fasta.gz or fasta.bz2)")
```

Comment:
It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details. *This comment may apply to more images, please consider checking all of them.*